### DISCUSSION TITLE
[Kavita] Display Localized title instead of Original title in the WebUI

### DISCUSSION THREAD

**Discussion by kayhos** [Kareadita/Kavita#3077] [Ideas]
### Idea Description

Hello,

I have been using Kavita for a short while but have a large amount of mangas, manwahs & al. Not being a native speaker I have a hard time remembering what each title is. Being able to display the localized named while browing with the Original name in subtitle would be greatly useful. 

I know that we can search both title types, but with a large collection, I don't always know what i'm looking for.

Simply entering the localized title in the original title and vice-versa is not suitable because the ComicInfo.xml files are read in another software and I believe the right data should be in the right field.

Thank you for taking the time to consider my proposal.

### Idea Category

Web UI

### Duration of Using Kavita

2 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
I do want to just chime in that Localized series name is a custom field for Kavita only. So the series name you choose is only your choice (like using official Japanese or English release).

**Reply by kayhos** [reply]
I'm not sure I understand your comment correctly. The localized name is customizable in Kavita, but I am loading it from the ComicInfo.xml file, so I am not changing it. 

I also think it is a bit weird to allow modification of the localized title but not the Japanese title, but this is another feature request.

**Reply by majora2007** [reply]
I'm saying the field in the ComicInfo for localizedSeries is only read for Kavita. I created it and updated Manga Manager since I couldn't get it done through the Anasai project.

**Comment by tntexploder1234** [comment]
@majora2007 I was about to post this same idea to be honest. A toggle in the library settings to display the custom-set localised name in the library would be very helpful. I store my manga in .cbz files and I can't edit their names without my manga downloader getting angry at me & I do kind of like having the romaji above the localised name.

https://imgur.com/a/YT9m1Wj

**Comment by majora2007** [comment]
In the latest nightly, we now have localized series name available for any series card on hover. I wonder if this solves the needs of this Feature Request:
![image](https://github.com/user-attachments/assets/193cb619-a8cd-47b1-b18c-a28b945685a7)

**Reply by tntexploder1234** [reply]
That looks exactly like how I'd like my library to look but mines kind of janked. I have kusuriya no hitorigoto as the original name with the localised name set to The Apothecary Diaries. I do ideally want my manga library to be fully sorted in english though, so that the localised name appears where the original one is if set and vice versa. Hoping for maybe a setting to switch their positions
<img width="165" alt="Screenshot 2024-08-21 at 17 10 14" src="https://github.com/user-attachments/assets/095e7a96-3532-4db3-b4eb-2eedb0372504">

**Reply by majora2007** [reply]
This happens because you have the series name in Japanese. But even with a setting, Kavita orders by sort name, which comes from the ComicInfo as well. I'm not sure how it could work to have a setting and 3 different values to use.

**Reply by tntexploder1234** [reply]
Ah I see, that is unfortunate. The comicinfo.xml in the cbz archive is automatically generated by my manga downloader and the titles come from the sites I download them from. I was planning on changing them but then it keeps losing track of what chapters it has already downloaded. Might try to create a separate folder and then move it into the library once I chance the comicinfo data

**Reply by majora2007** [reply]
You might be able to use komf to tag it properly as komf has the ability to write to comicinfo.xml and can set the original language vs localized.
